src/extensions/default/Inline3DParametersEditor/ParameterEditor.js

```diff
@@ -0,0 +1,149 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var Strings            = brackets.getModule("strings"),
+        Mustache           = brackets.getModule("thirdparty/mustache/mustache");
+
+    /** Mustache template that forms the bare DOM structure of the UI */
+    var Template = require("text!ParameterEditorTemplate.html");
+
+     //used to control the smoothness of scroll.
+     var MODIFIER = 10;
```

Comment:
Can you add a unit to this so it's clear that this refers to?  `MODIFIER_PX` maybe?